src/index.html

```diff
@@ -3,10 +3,402 @@
+
+        <section class="stars">
+          <div class="stars__block stars__block--4">
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+          </div>
+
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <p class="card__price-title">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </section>
+
+        <div class="card__button">
+          <a href="#" class="card__button-link">
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image">
+          <img
+            class="card__image-picture"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+        </div>
+
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <section class="stars">
+          <div class="stars__block stars__block--4">
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+          </div>
+
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <p class="card__price-title">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </section>
+
+        <div class="card__button">
+          <a href="#" class="card__button-link">
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image">
+          <img
+            class="card__image-picture"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+        </div>
+
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <section class="stars">
+          <div class="stars__block stars__block--4">
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+
+            <div class="stars__block-item"></div>
+          </div>
+
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <p class="card__price-title">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </section>
+
+        <div class="card__button">
```

Comment:
you may get rid of this wrapper
just apply all styles you need direct to link (`card__button-link` now, but you may rename it after removing wrapper)